modules/installing-python-packages-on-your-code-server-workbench.adoc

```diff
@@ -0,0 +1,70 @@
+:_module-type: PROCEDURE
+
+[id="installing-python-packages-on-your-code-server-workbench_{context}"]
+= Installing Python packages on your code-server workbench
+
+[role='_abstract']
+You can install Python packages that are not part of the default workbench image by adding the package and the version to a `requirements.txt` file and then running the `pip install` command in a terminal window.
+
+ifndef::upstream[]
+NOTE: You can also install packages directly, but {org-name} recommends using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks. In addition, using a `requirements.txt` file is also useful when using a S2I build to deploy a model.
+endif::[]
+ifdef::upstream[]
+NOTE: You can also install packages directly, but using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks is recommended. In addition, using a `requirements.txt` file is also useful when using a S2I build to deploy a model.
+endif::[]
+
+.Prerequisites
+* A launched and running code-server workbench.
+
```

Comment:
is having pip installed also a prereq?

Reply:
all our workbench images have pip installed, so it should not be necessary to mention this;

I used to think that prerequisites are important for long tutorials, when the body of the procedure boils down to running a single terminal command, it's not so useful, me thinks. https://glebbahmutov.com/blog/common-documentation-mistakes-and-how-to-avoid-them/#the-documentation-guides